### PR TITLE
Yoda delete persistence no top-level methods.

### PR DESCRIPTION
**Pull request by dkorolev**
Hi @mzhurovich && @sompylasar ,

Amending https://github.com/c5t/Current/pull/245 and removing top-level methods [altogether](http://www.oxforddictionaries.com/us/translate/german-english/n%C3%A4her).

(( Didn't help CTFO though. On it. ))

Thanks,
Dima

### CODE REVIEW

File: Yoda/test.cc [hunk 1]
```diff
@@ -54,13 +59,23 @@ TEST(Yoda, WritesToFile) {
...
   while (bricks::FileSystem::GetFileSize(persistence_file_name) != yoda_golden_data.size()) {
     ;  // Spin lock.
   }
-
   EXPECT_EQ(yoda_golden_data, bricks::FileSystem::ReadFileAsString(persistence_file_name));
+
+  api.Transaction([](PersistingAPI::T_DATA data) {
+    Dictionary<YodaEntryToPersist>::Mutator(data).Delete("one");
+  }).Wait();
+  while (bricks::FileSystem::GetFileSize(persistence_file_name) != yoda_golden_data_after_delete.size()) {
```
Comment (sompylasar) [inline Yoda/test.cc]: @dkorolev  Please see my comment to this line in one of the previous PRs.

Comment (dkorolev) [inline Yoda/test.cc]: Replied there :)

Comment (sompylasar) [inline Yoda/test.cc]: Ah, see (went through emails sequentially). How about checking `> 0` instead? The exact value (and, automagically, its length) will be checked below.

Comment (dkorolev) [inline Yoda/test.cc]: I think I tried that before and it didn't work for some reason.

This code is going away soon regardless.

And, as always, have a nice day!
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783

Comment (sompylasar) [inline Yoda/test.cc]: OK, keep it this way then...

**Comment by dkorolev** [comment]
@mzhurovich Gentle ping?

**Comment by mzhurovich** [comment]
@dkorolev LGTM! Sorry for the delay - I completely forgot about this PR :(

**Pull request merged**